### PR TITLE
Add types for random-web-token

### PR DESCRIPTION
**Pull request by schrudolf**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/random-web-token/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ * @param {number} length same length as the generated Token
+ *
+ * @param {string} token The received token from genSync() or genAsync()
+ *
+ * @param {string | undefined} allowedPlusCharacters (This is optional) extra allowed characters in string -> "!%/"
+ *
+ * @example asyncValidator("extra", 50, token, "")
+ * @returns {Promise<boolean>}
+ */
+declare function asyncValidator(type: Types, length: number, token: string, extraCharacters?: string): Promise<boolean>;
```
Comment (peterblazejewicz) [inline types/random-web-token/index.d.ts:80]:
```suggestion
declare function asyncValidator(type: Types, length: number, token: string, allowedPlusCharacters?: string): Promise<boolean>;
```
looks like name mismatch here

**Comment by peterblazejewicz** [comment]
Can you please run prettier formatter on your code?
https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes
```bash
npm run prettier -- --write types/random-web-token/*.ts
```
the JSDocs comments seem to be misformatted, thanks!

File: types/random-web-token/index.d.ts (hunk 1 above)
Comment (schrudolf) [reply types/random-web-token/index.d.ts]: I corrected. Thanks

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@schrudolf 
you can also publish those types alongside your own module directly, relevant docs:
https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package
Thanks!

**Comment by schrudolf** [comment]
Ready to merge

**Pull request merged**